src/AstGen.zig

```diff
@@ -5070,14 +5070,14 @@ fn structDeclInner(
                 var error_notes = std.ArrayList(u32).init(astgen.arena);
 
                 for (record.items[1..]) |duplicate| {
-                    try error_notes.append(try astgen.errNoteTok(duplicate, "other field here", .{}));
+                    try error_notes.append(try astgen.errNoteTok(duplicate, "duplicate field here", .{}));
                 }
 
                 try error_notes.append(try astgen.errNoteNode(node, "struct declared here", .{}));
 
                 try astgen.appendErrorTokNotes(
                     record.items[0],
-                    "duplicate struct field: '{s}'",
```

Comment:
Relating to first comment: okay, fair enough, this was inconsistent between inits vs type decls before. I think the best option is to *not* have the name since, as discussed above, it's redundant information. Also, `duplicate struct field` is a much clearer message than `struct field name conflict`; I'm not sure why you felt the need to change that bit.